app/src/test/java/io/github/droidkaigi/confsched2017/util/RxTestSchedulerRule.kt

```diff
@@ -5,7 +5,7 @@ import io.reactivex.plugins.RxJavaPlugins
 import io.reactivex.schedulers.TestScheduler
 import org.junit.rules.ExternalResource
 
-class RxTestSchedulerRule : ExternalResource() {
+object RxTestSchedulerRule : ExternalResource() {
```

Comment:
TestScheduler instance must be singleton.
Now `RxTestSchedulerRule` is used from multiple test class, so this must be `object`.